src/components/navbars/MobNav.jsx

```diff
@@ -0,0 +1,15 @@
+import joinIcon from '../../assets/icons/register.png';
```

Comment:
aria-labels for buttons.

Reply:
@vrudnitskaya apologies no need to add aria-labels if there is visible text. Thanks for pointing that out @AnnaPestova1 

[https://kb.iu.edu/d/axgb](url)

Reply:
This link is not working

Reply:
[https://kb.iu.edu/d/axgb](url)

Reply:
https://kb.iu.edu/d/axgb